<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_dialog</class>
 <widget class="QDialog" name="settings_dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>137</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Where to open explorer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="explorerDefaultPathEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="explorerDefaultPathSelectBtn">
       <property name="styleSheet">
        <string notr="true">padding: 2px 4px;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="showConversionWarningDialogBox">
     <property name="text">
      <string>Show warning dialog while starting the conversion</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useSystemExplorerBox">
     <property name="text">
      <string>Use system explorer for opening files and folders</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveSettingsBtn">
     <property name="styleSheet">
      <string notr="true">padding: 4px 0;</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
